debian/changelog

```diff
@@ -24,7 +25,13 @@ cloud-init (19.4-51-g8409dd7d-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
     - util: rename get_architecture to get_dpkg_architecture (#173)
     - Ensure util.get_architecture() runs only once (#172)
 
- -- Ryan Harper <ryan.harper@canonical.com>  Wed, 29 Jan 2020 12:09:51 -0600
+  [ Daniel Watkins ]
```

Comment:
Not a firm objection, but cloud-init changelogs generally use author attribution as a trailing [<you>] at the end of entries .... and for cloud-init upstream core maintainers, we generally redact the authorship attribution.